develop/docker-compose/docker-compose.linux.yml

```diff
@@ -12,3 +12,5 @@ services:
     environment:
       - "TEMPORAL_GRPC_ENDPOINT=localhost:7233"
     network_mode: host
+    ports:
```

Comment:
@alexshtin this docker-compose file doesn't mention image targets. How does it work/used?

Reply:
@feedmeapples this file extends from [the base compose file](https://github.com/temporalio/temporal/blob/master/develop/docker-compose/docker-compose.yml)
Developers run these commands in development environment: https://github.com/temporalio/temporal/blob/master/Makefile#L410-L414

Reply:
oh thanks!